src/App.js

```diff
@@ -23,6 +21,25 @@ import { Footer, Navbar } from "./components/molecules";
 import { ScrollToTop } from "./components/atoms";
 //> Routes
 import Routes from "./Routes";
+//> Actions
+import {
+  ferry,
+  login,
+  logout,
+  fetchGitLabServers,
+  appendSourceObjects,
+  getAllPageUrls,
+  getData,
+  saveSettings,
+  register,
+  readCache,
+  updateCache,
+  writeCache,
+  getAllTalks,
+  getTalk,
+  uploadTalk,
+  deleteTalk,
+} from "./actions";
```

Comment:
I would split this up in mutliple categories.

For instance, you can create a group for talks. This eases the mind and improve readability.

Reply:
Yep. I agree. I suggest following structure:

**For all action sections:**

```
import {
  getAllTalks,
  getTalk,
  uploadTalk,
  deleteTalk,
} from "actions/intel/talksActions";
```

**For the action core:**

`import { intelFerry } from "actions/core"`

- As you can see I changed `ferry` to `intelFerry` in order to prevent naming conflicts for future endpoints like "intel".
- Rename `intel` to `intel`-> every front folder is lowercase
- Adjust the import structure, as mentioned by @Aichnerc.

Reply:
Can we have a meeting about this before changing the structure?